api/repositories/org_repository.go

```diff
@@ -205,6 +205,21 @@ func (r *OrgRepo) GetOrg(ctx context.Context, info authorization.Info, orgGUID s
 	return orgRecords[0], nil
 }
 
+func (r *OrgRepo) GetOrgForDeletion(ctx context.Context, info authorization.Info, orgGUID string) (OrgRecord, error) {
```

Comment:
This name describes the case where we use this function, but not what it's actually doing. There may be other cases where we need this same functionality for non-deletion uses. Perhaps `GetOrgIgnoringCFRoles` is more descriptive of what's happening?

Reply:
I've changed the name to `GetOrgUnfiltered`.